src/check.ml

```diff
@@ -1,11 +1,11 @@
-(* 
+(*
```

Comment:
It makes the actual content of diffs hard to read if there's a lot of unrelated whitespace changes. Better do those things outside of feature branches.

Reply:
Github luckily has an option to ignore it, but on the cli it's annoying.

Reply:
Sorry!
I had it in a separate commit for that reason, but did not think about the code review process :-1: